src/foam/nanos/boot/Boot.java

```diff
@@ -41,7 +41,7 @@ public Boot(String datadir) {
     XLocator.set(root_);
 
     Logger logger = new ProxyLogger(new StdoutLogger());
-    root_.put("logger", logger);
+    root_ = root_.put("logger", logger);
```

Comment:
Why is this necessary? root_ is a ProxyX, which does this automatically.

Reply:
updated.